addon/services/segmentio.js

```diff
@@ -0,0 +1,29 @@
+import Ember from 'ember';
+
+// Should match methods in snippet-core.min.js
+const SEGMENT_METHODS = [
```

Comment:
Am doing that now 49 properties so not terrible; however [stupid phantomjs](https://groups.google.com/forum/#!msg/phantomjs/r0hPOmnCUpc/uxusqsl2LNoJ)